config/config/wc-config.yaml

```diff
@@ -144,10 +148,14 @@ fluentd:
     - effect: NoSchedule
       key: node-role.kubernetes.io/master
       value: ""
+    - key: node-role.kubernetes.io/control-plane
+      effect: NoSchedule
+      value: ""
 
   ## Only run on master nodes.
   nodeSelector:
     node-role.kubernetes.io/master: ""
+    node-role.kubernetes.io/control-plane: ""
```

Comment:
Could you change this into affinity instead? AFAIK node selector requires nodes to have all of the specified labels. So with this we require nodes to have both `master` and `control-plane` labels, otherwise it will not be scheduled there.

Reply:
ok 
I do not remember the reason for keeping the nodeSelector here instead of affinity